### DISCUSSION TITLE
Stop apps

### DISCUSSION THREAD

**Discussion by TechieFsn** [d4rken-org/sdmaid-se#1409] [Q&A]
**Is your feature request related to a problem? Please describe.**

I would love a feature to automatically scan for all user installed applications and force stop all selected apps (leaving unselected applications alone).  I subscribe to another application that does this, as well as clean cache, but your application is vastly superior except for missing this one batch force stop feature.

Appreciate your consideration, and for making a great application!

**Comment by d4rken** [comment]
You can already do this in the AppControl tool.
